src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,14 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
+        object System.Collections.ICollection.SyncRoot { get { return default(object); } }
```

Comment:
Maybe you need `protected override` modifiers etc. since it is overriding an existing defined class/interface.